src/utils/hact/update-job.js

```diff
@@ -0,0 +1,27 @@
+export const buildUpdateJob = (existingTasks, addedTasks, reference) => {
+  const allTasks = [...existingTasks, ...addedTasks]
+  return {
+    relatedWorkOrderReference: {
+      id: reference,
+      description: 'string',
+      allocatedBy: 'string',
+    },
+    typeCode: 8,
+    moreSpecificSORCode: {
+      rateScheduleItem: allTasks.map((task) => {
+        let codeAndName = task.code.split(' - ')
+        let showDescription =
+          task.description &&
+          task.description.length > 0 &&
+          task.description !== 'string'
```

Comment:
A bit confused of what we are trying to do here

Reply:
by mistake I added value `"string"` to some tasks, so some might come with description `"string"`